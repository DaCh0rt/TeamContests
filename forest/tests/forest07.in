3 508
0 0
1 1
2 2
