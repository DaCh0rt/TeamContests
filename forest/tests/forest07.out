3197.51 813605.65
